source/GameInterface/Services/Settlements/Patches/SettlementFieldPatches.cs

```diff
@@ -0,0 +1,86 @@
+﻿using GameInterface.AutoSync;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.CampaignSystem.Actions;
+using TaleWorlds.CampaignSystem.CampaignBehaviors;
+using TaleWorlds.CampaignSystem.Party.PartyComponents;
+using TaleWorlds.CampaignSystem.Settlements;
+
+namespace GameInterface.Services.Settlements.Patches
+{
+    internal class SettlementFieldPatches : IAutoSync
+    {
+        public SettlementFieldPatches(IAutoSyncBuilder autoSyncBuilder)
```

Comment:
For now I am requiring a test for every field and property. Could you please add them